features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Wizard.jsx

```diff
@@ -308,13 +310,27 @@ class Wizard extends React.Component {
         this.setState({ value: event.target.value });
     };
 
+    /**
+     *
+     * Set state.redirect to true to redirect to the API console page
+     * @memberof Wizard
+     */
+    handleRedirectTest = () => {
```

Comment:
Shall we use the class method definition format, not this class member function style.
We have used this style before but don't recommend to use it anymore.
like

```javascript
class A {
    constructor() {
        this.myMethod = this.myMethod.bind(this);
    }

myMethod() {}
}
```

Reply:
Fixed